cobalt/browser/BUILD.gn

```diff
@@ -83,3 +85,22 @@ action("embed_polyfilled_javascript") {
     rebase_path("embedded_resources", root_build_dir),
   ]
 }
+
+source_set("metrics") {
+  # TODO(cobalt, b/375655377): remove testonly declaration, needed because of
+  # being depended-on by :browser above.
+  testonly = true
```

Comment:
Does testonly prevent this from being included in non-testonly targets? This is generally how testonly targets are treated within Google.